length: 4.97
width: 1.6
heigth: 1.4
wheel_base: 2.97
rear_overhang: 1.08 #后悬
front_overhang: 0.92 #前悬
tire_radius: 0.32